corehq/ex-submodules/couchexport/export.py

```diff
@@ -338,6 +338,7 @@ def _wrap_row(first_entry, rows):
                 first_entry = next(first_entry_rows)
                 wrapped_rows = name, (cls(row) for row in _wrap_row(first_entry, rows_to_wrap))
             else:
+                rows = list(rows)
                 rows = _wrap_row(rows[0], list(rows))
```

Comment:
nit: you can drop the second call to `list(rows)`

Reply:
https://github.com/dimagi/commcare-hq/pull/23782/commits/72192e89865204e769395647f278d0662b6f8b07